lib/improver/tests/wind_calculations/wind_components/test_ResolveWindComponents.py

```diff
@@ -238,8 +238,8 @@ def test_wind_from_direction(self):
         self.wind_direction_cube.rename("wind_from_direction")
         ucube, vcube = self.plugin.process(
             self.wind_speed_cube, self.wind_direction_cube)
-        self.assertTrue(np.allclose(ucube.data, expected_u, atol=1e-6))
-        self.assertTrue(np.allclose(vcube.data, expected_v, atol=1e-6))
+        self.assertArrayAllClose(ucube.data, expected_u, atol=1e-5)
+        self.assertArrayAllClose(vcube.data, expected_v, atol=1e-5)
```

Comment:
With the change of testing method to the iris's assertArrayAllClose, why does this mean the precision has changed.

Reply:
The code originally specified an absolute tolerance of 1e-6, which is a tighter tolerance than the 5 decimal places described in the comment. The change to absolute tolerance of 1e-5 matches with the comment and is less picky about the last decimal place of the expected arrays.

Changing the assertion to assertArrayAllClose (inherited from iris) produces better test output than a True/False result from np.allclose.